tests/model_serving/model_runtime/vllm/cpu/ibm_power_z/utils.py

```
@@ -0,0 +1,43 @@
+    messages: list[dict[str, str]],
+    max_tokens: int,
+) -> dict[str, Any]:
+    """Send a POST request to /v1/chat/completions matching the OpenAI chat API."""
+    url = f"{get_exposed_isvc_url(isvc=isvc)}{OpenAIEnpoints.CHAT_COMPLETIONS}"
+    payload = {
+        "model": isvc.instance.metadata.name,
+        "messages": messages,
+        "max_tokens": max_tokens,
+    }
+    LOGGER.info("Sending chat completions request", url=url, payload=payload)
+    response = requests.post(url, headers=RestHeader.HEADERS, json=payload, verify=False)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)